README.md

```diff
@@ -1,8 +1,107 @@
+Table of Contents
+=================
 
+* [Introduction](#introduction)
+* [Charts repository](#repository)
+  * [Available Charts in Repository](#available-charts-in-repository)
+  * [Usage](#usage)
+  * [Development](#development)
+  * [Testing](#testing)
+* [Templates](#templates)
+  * [Role-Based Access Control](#rbac)
 
-## How to play
+# Introduction
+
+This repository contains Helm templates and charts used by other applications part of the Snowdrop team.
```

Comment:
To be reviewed please

Reply:
What do you mean?

Reply:
You should also include pat of the introduction that this project also:
- Serves as Helm index repository
- Contains  best practices to design Helm charts (eg. RBAC, ...)

Reply:
Changed